vision/dcgan_mnist/dcgan_mnist.jl

```diff
@@ -5,6 +5,7 @@ using Flux.Losses: logitbinarycrossentropy
 using Images
 using MLDatasets
 using Statistics
+using Distributions:Normal
```

Comment:
```suggestion
```

Reply:
Is it necessary to import Distributions?

Reply:
I deleted the import because we aren't using `Normal` anymore. `randn` comes as part of Base.